### PR TITLE
Return a list of project targets excluding aggregate targets.

### PR DESCRIPTION
**Pull request by segiddins**
Closes https://github.com/CocoaPods/CocoaPods/issues/2169.

\c @kylef

### CODE REVIEW

**Comment by kylef** [comment]
My understanding of Xcodeproj internals is a little non-existent so I'm no good to review this one.

But should there be tests / changelog entry?

**Comment by segiddins** [comment]
Yes there should be, but this PR was based off of someone else's branch.

File: lib/xcodeproj/project.rb [hunk 1]
```diff
@@ -490,13 +490,22 @@ def reference_for_path(absolute_path)
...
     #
     def targets
       root_object.targets
     end
 
+    # @return [ObjectList<PBXNativeTarget>] A list of all the targets in the
+    #         project excluding aggregate targets.
+    #
+    def native_targets
+      root_object.targets.reject do |target|
```
Comment (mrackwitz) [inline lib/xcodeproj/project.rb]: Should use `select` in combination with `target.is_a? PBXNativeTarget`.

**Comment by segiddins** [comment]
@mrackwitz do you want to clean this PR up and get it merged?

**Pull request closed without merging**